test/basicTactics.lean

```diff
@@ -4,6 +4,7 @@ example : ∀ a b : Nat, a = b → b = a := by
   introv h
   exact h.symm
 
+#check Nat
```

Comment:
I'm guessing that the enforcement is currently broken; perhaps you need to capture stderr instead?

Reply:
Yes, it's strange --- `make test` fails locally! I'll push a minor change (better file names that won't be clobbered by parallel tests) and try again.

Reply:
Successful failure now!